runtime/vara/src/lib.rs

```diff
@@ -973,6 +973,9 @@ parameter_types! {
     pub const DispatchHoldCost: u64 = 100;
 
     pub const OutgoingLimit: u32 = 1024;
+    // 64 MB, must be less than max runtime heap memory.
+    // NOTE: currently runtime heap memory is 1 GB (see https://shorturl.at/DET45)
+    pub const OutgoingBytesLimit: u32 = 64 * 1024 * 1024;
```

Comment:
You opinion here is welcome. May be we should set bigger limit, may be smaller. Actually current memory is 32 mb, so I think is impossible to send more than that value, because of gas consumption&

Reply:
Yes 512 it's too big currently. We must have at lease 2 GB runtime heap size for that by default.